cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala

```diff
@@ -140,12 +152,38 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
           statsOnlyResultTable()
         }
         p("For more details on primary and secondary server roles, see <<operations-manual#clustering-introduction-operational, Cluster overview>>.")
+        note {
+          p("The `TOPOLOGY` are only available to standard databases and not composite databases.")
+        }
+      }
+      section("Creating composite databases", "administration-databases-create-composite-database", "enterprise-edition") {
+        // TODO: add a link to the place explaining what composite databases are
+        p(
+          """Composite databases are a type of database that instead of containing data holds other databases though aliases.""".stripMargin)
+        p("Composite databases can be created using `CREATE COMPOSITE DATABASE`.")
+        query("CREATE COMPOSITE DATABASE inventory", ResultAssertions(r => {
+          assertStats(r, systemUpdates = 1)
+        })) {
+          statsOnlyResultTable()
+        }
+        note {
+          p("""Composite database names are subject to the same rules as standard databases.
+              |One difference is however that both dots and dashes needs to be enclosed within backticks, not just dashes.""")
```

Comment:
```suggestion
              |One difference is however that the deprecated syntax using dots without enclosing the name in backticks is not available. Both dots and dashes needs to be enclosed within backticks when using composite databases.""")
```
Probably should say this is deprecated.